eduvpn/__init__.py

```diff
@@ -5,7 +5,4 @@
 
 import pkg_resources
 
-try:
-    __version__ = pkg_resources.require("eduvpn")[0].version # type: str
-except pkg_resources.DistributionNotFound:
-    __version__ = "0.0dev" # type: str
+__version__ = pkg_resources.require("eduvpn")[0].version # type: str
```

Comment:
this is not right, that try except is there for a reason, if one uses the module but the eduvpn is not completely installed it doesn't know the version, so in that case it returns `0.0dev`. If mypy gives an error here then that is new (a new version of mypy) and that error should be muted.